client/tasks/fills/Marketing/Marketing.scss

```diff
@@ -17,4 +17,19 @@
 			color: $gray-700;
 		}
 	}
+
+	.woocommerce-plugin-list__title {
+		background: inherit;
+		padding-top: $gap-larger;
+		padding-bottom: $gap;
+
+		h3 {
+			text-transform: uppercase;
+		}
+	}
+
+	.woocommerce-plugin-list__plugin {
+		border-top: none;
+		border-bottom: 1px solid #e0e0e0;
+	}
```

Comment:
I believe the correct place for these css changes is actually in `PluginList.scss`.